.github/workflows/wkdev-sdk.yml

```diff
@@ -36,20 +36,20 @@ jobs:
           remove-haskell: true
           remove-codeql: true
           remove-docker-images: true
-      
+
       - name: Install podman
         uses: gacts/install-podman@v1
 
-      - name: Install fuse-overlayfs
-        run: sudo apt-get -y install fuse-overlayfs
+      - name: Install fuse-overlayfs and qemu-user-static
+        run: sudo apt-get -y install fuse-overlayfs qemu-user-static
 
       - name: Checkout repo
         uses: actions/checkout@v4
 
       - name: Build image
         run: |
           source ./register-sdk-on-host.sh
-          wkdev-sdk-bakery --mode=build --verbose
+          wkdev-sdk-bakery --mode=build --platforms=linux/arm/v7,linux/amd64 --verbose
```

Comment:
Could we actually make this build both in parallel?

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Will try that, great suggestion.

Reply:
That requires a rework of the wkdev-sdk-bakery if we want that.
We need to move 'image manifest' creation out of wkdev-sdk-bakery, execute that first, then build image for platform A, then for platform B, and each time register the image within the manifest. Then finally push the manifest, together with the images, building a single multi-arch container including both architectures, under the same name/tag.

IMHO I'd leave it as-is, until we really see the image build time as an issue.